wpical/src/main/native/include/fieldmap.h

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+#pragma once
+
+#include <numbers>
+#include <vector>
+
+#include <Eigen/Dense>
+#include <Eigen/Geometry>
+#include <tagpose.h>
+#include <wpi/json.h>
+
```

Comment:
The only headers used here are `<vector>`, `<tagpose.h>`, and `<wpi/json.h>`. `<numbers>` can be swapped with `<cmath>` for `std::fmod`. Everything else can be removed.